FND/Places/Addresses.rdf

```diff
@@ -255,36 +258,18 @@
 	<owl:Class rdf:about="&fibo-fnd-plc-adr;PhysicalAddress">
 		<rdfs:subClassOf rdf:resource="&fibo-fnd-plc-adr;Address"/>
 		<rdfs:subClassOf>
-			<owl:Class>
-				<owl:unionOf rdf:parseType="Collection">
-					<owl:Restriction>
-						<owl:onProperty rdf:resource="&fibo-fnd-plc-adr;hasPostalCode"/>
-						<owl:onDataRange rdf:resource="&rdfs;Literal"/>
-						<owl:qualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">1</owl:qualifiedCardinality>
-					</owl:Restriction>
-					<owl:Restriction>
-						<owl:onProperty rdf:resource="&fibo-fnd-plc-adr;hasIndividualPostcode"/>
-						<owl:onClass rdf:resource="&fibo-fnd-plc-adr;Postcode"/>
-						<owl:qualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">1</owl:qualifiedCardinality>
-					</owl:Restriction>
-				</owl:unionOf>
-			</owl:Class>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-plc-adr;hasIndividualPostcode"/>
+				<owl:onClass rdf:resource="&fibo-fnd-plc-adr;Postcode"/>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
```

Comment:
why not have max of 1? Can an address have many postCodes?

Reply:
I've added that back.